Dockerfile.api-gateway

```diff
@@ -1,12 +1,13 @@
 # Étape de build
-FROM node:18-alpine AS builder
+FROM node:22-alpine AS builder
```

Comment:
I've change node version because of requirement node version greater than 18 or 20 by some dependencies waring in the running